### DISCUSSION TITLE
Case sensitive in name

### DISCUSSION THREAD

**Discussion by jenny-hm-lee** [microsoft/presidio#1405] [Q&A] [answered]
I have tried the AnalyzerEngine()  with the following two text and get different answers.  The only difference between them is their upper vs lower cases.

1. text = 'THOMAS, PATRICIA JANE'
2. text = "Thomas, Patricia Jane"

How can I correctly detect the person name in text 1?

**Code:**
`from presidio_analyzer import AnalyzerEngine

text = 'THOMAS, PATRICIA JANE'
analyzer = AnalyzerEngine()
analyzer_results = analyzer.analyze(text=text, language="en")
print(text)
print(analyzer_results)
print('-----')

text = "Thomas, Patricia Jane"
analyzer = AnalyzerEngine()
analyzer_results = analyzer.analyze(text=text, language="en")
print(text)
print(analyzer_results)
print('-----')`

**Result**
![image](https://github.com/microsoft/presidio/assets/4551564/b0f97385-5bc1-4972-b9f2-8669c46ba32c)

**Comment by omri374** [comment, marked as answer]
Hi, the default spaCy model has limited accuracy on uppercase and lowercase words. I would suggest to try a few model examples on the demo website and change the underlying model. There are lots of examples showing how to do this.

https://huggingface.co/spaces/presidio/presidio_demo

**Reply by jenny-hm-lee** [reply]
thanks
